code/game/objects/items/weapons/material/knives.dm

```diff
@@ -25,6 +25,20 @@
 			M = user
 		return eyestab(M,user)
 
+/obj/item/weapon/material/knife/verb/extract_shrapnel(var/mob/living/carbon/human/H)

```

Comment:
so you can do this with a kitchen knife? Tactical knives made more sense balance-wise

Reply:
i think it's fine for all knives, really, any material knife works atm

Reply:
Made it more painful via actual dmg